### ISSUE TITLE
Text structure and display

### ISSUE THREAD

**Issue by jedwardwalters** [srophe/syriac-corpus#48]
@davidamichelson @dlschwartz @nathangibson @wsalesky 
I edited three different kinds of texts (one demonstration, one poetic memra, and one scripture) to remove the div elements like we discussed yesterday.

The three files are:
https://github.com/srophe/syriac-corpus/blob/master/data/tei/1.xml
https://github.com/srophe/syriac-corpus/blob/master/data/tei/27.xml
https://github.com/srophe/syriac-corpus/blob/master/data/tei/100.xml

**Comment by davidamichelson** [comment]
Thanks, for https://github.com/srophe/syriac-corpus/blob/master/data/tei/100.xml can you change even the verses to milestones instead of the tei:ab

**Comment by davidamichelson** [comment]
Just to clarify, keep the ab but put the verse numbers into milestones

**Comment by jedwardwalters** [comment]
@davidamichelson 
Okay, done. See pull request #49 
https://github.com/srophe/syriac-corpus/blob/master/data/tei/100.xml

**Comment by wsalesky** [comment]
@jedwardwalters Can you check 100.xml I am getting a well formed error. Thanks!

**Comment by jedwardwalters** [comment]
@wsalesky Is it possible that the problem here is that <ab> is not allowed within a paragraph element? Maybe I'm misreading this, but now that I'm looking more closely, it doesn't seem like <ab> can go inside of a p.
http://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-ab.html

**Comment by wsalesky** [comment]
No, I think the problem is that the <p> is not closed. eXist does not
validate, it just checks for wellformedness (and valid xml:ids)

**Comment by jedwardwalters** [comment]
Sorry. I literally just saw that missing closed p element. Fixing now.

**Comment by jedwardwalters** [comment]
@wsalesky Okay, I fixed the missing p element in 100.xml.

I also added back in head elements in 1.xml for the section numbers--we discussed this in the editors meeting today.

**Comment by jedwardwalters** [comment]
@dlschwartz @davidamichelson 
Did you guys make any decisions today after I had to take off? I could join the meeting tomorrow with Winona for a few minutes if that would be helpful.

**Comment by dlschwartz** [comment]
@jedwardwalters I'll add you to the meeting this morning.

**Comment by jedwardwalters** [comment]
One additional note: in my research I also confirmed that the ab tag is not supposed to be within a p element; it's only supposed to be in a div.

**Comment by dlschwartz** [comment]
Thanks @jedwardwalters. I have been thinking that we should just bite the bullet and assert a canonical (at least corpus-canonical) text structure. I am uneasy about not having text in any meaningful text node, especially since I really like the idea of pulling texts into SPEAR factoid pages. I was thinking that this might be worth it even if we lost the ability to offer multiple views. If Jamey's proposal still accommodates offering multiple views and still allows linking to the various different milestones as well, I think this might be a win-win. It's not a violation of xml's inability to allow overlapping structures to simply have a view option that returns all the text between two milestones. Does this raise any concerns for you @wsalesky? @davidamichelson?

**Comment by davidamichelson** [comment]
Digital Mishna is a solid project that has probably thought things through,
so that is good. I guess that would be a good solution use divs for the
canonical versions but then allow milestones so that at least there are
other Tables of Contents possible. LEt's discuss one more time on Tuesday.

**Comment by nathangibson** [comment]
This makes sense to me too. I would think that marking canonical divisions using text-containing nodes would help not only with SPEAR but also with CTS down the road. 

A couple clarifications to discuss: 
 1. If a text doesn't have any truly canonical divisions, do we simply create them?
 2. Is it the case that alternate structures will serve to display numbers in the text and links from the table of contents, but will _not_ be used to retrieve such text sections? Because I imagine retrieving "page 10" as a text node might be difficult if the structure is as follows?
```
<milestone unit="page" n="10"/>
blah
<p>
blah
<milestone unit="page" n="11"/>
blah
</p>
```

**Comment by dlschwartz** [comment]
I would be fully satisfied to retrieve only by canonical text structure if one can still link to other milestones.

If no text structure exists, (say in an unedited text that exists in only a small number of mss) we can simply make a preferred available numbering (in this example one set of folio numbers) the canonical numbering. When someone decides to take that unedited transcription of an ms and make it into an actual digital edition, they would hopefully be able to apply a better canonical structure. They would also likely want to retain the folio numbers in milestone elements.

In other words, in the short run, as much as we might want to be in the business of universally applying some kind of structure to our texts, we will need instead to take what we have and put it up with the best available structure adopted as the provisional, if not final, "canonical" structure.

**Comment by jedwardwalters** [comment]
Thanks, everyone, for weighing in. We can follow up on Tuesday, and I'll hold off on any more changes until then.

**Comment by davidamichelson** [comment]
Closing this, see the resolution at https://github.com/srophe/syriac-corpus-app/issues/41
